app/src/main/res/values/strings.xml

```diff
@@ -560,12 +560,25 @@ Upload your first media by tapping on the add button.</string>
   <string name="dialog_box_text_nomination">Why should %1$s be deleted?</string>
   <string name="review_is_uploaded_by">%1$s is uploaded by: %2$s</string>
   <string name="default_description_language">Default description language</string>
-  <string name="delete_reason_spam_selfie">A selfie</string>
-  <string name="delete_reason_spam_blurry">Blurry</string>
-  <string name="delete_reason_spam_nonsense">Nonsense</string>
-  <string name="delete_reason_spam_other">Other</string>
-  <string name="delete_reason_copyright_pressphoto">Press photo</string>
-  <string name="delete_reason_copyright_internetphoto">Random photo from internet</string>
-  <string name="delete_reason_copyright_logo">Logo</string>
-  <string name="delete_reason_copyright_other">Other</string>
+  <string name="delete_helper_make_deletion_toast_1">Trying to nominate</string>
+  <string name="delete_helper_make_deletion_toast_2">for deletion </string>
+  <string name="delete_helper_delete_summary_1">Nominating</string>
+  <string name="delete_helper_delete_summary_2">for deletion.</string>
+  <string name="delete_helper_show_deletion_notification_title">Nominating for Deletion</string>
+  <string name="delete_helper_show_deletion_notification_title_if">: Success</string>
+  <string name="delete_helper_show_deletion_notification_message_if_1">Successfully nominated</string>
+  <string name="delete_helper_show_deletion_notification_message_if_2">: deletion.</string>
+  <string name="delete_helper_show_deletion_notification_title_else">: Failed</string>
+  <string name="delete_helper_show_deletion_notification_message_else">Could not request deletion.</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_spam_selfie">A selfie</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_spam_blurry">Blurry</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_spam_nonsense">Nonsense</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_spam_other">Other</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_copyright_press_photo">Press photo</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_copyright_internet_photo">Random photo from internet</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_copyright_logo">Logo</string>
+  <string name="delete_helper_ask_reason_and_execute_reason_copyright_other">Other</string>
+  <string name="delete_helper_ask_reason_and_execute_alert_set_positive_button">Ok</string>
```

Comment:
I am sure we have Ok and Cancel strings in our file already. Please do not add them again, it is duplicate.